torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -11522,7 +11522,7 @@ TEST_F(NVFuserTest, FusionNonUniqueBroadcastSize_CUDA) {
   fusion.addInput(tv1);
   fusion.addInput(tv2);
 
-  auto tv3 = broadcast(tv0, {false, true});
+  auto tv3 = broadcast(tv0, {true, false});
```

Comment:
When doing `{false, true}`, the `inlineAt` will try to set CA position to 1 instead of 2 due to hoisting, which will not be a problem.